doc/doxygen/src/build-system-basics.md

```diff
@@ -0,0 +1,56 @@
+Build System Basics                                       {#build-system-basics}
+============================
+
+Variables declaration guidelines              {#variable-declaration-guidelines}
+============
+
+export (export OUTPUT = $(shell some-command))
```

Comment:
compiled to HTML these headlines look a bit ugly, I would move the sample command below into the section body.

Reply:
addressed